oci/zookeeper/documentation.yaml

```diff
@@ -18,6 +18,41 @@ description: |
   whereas from version 3.8.2 onward the images are now ROCKs. As such the
   entrypoint is now Pebble. Read more on the
   [Rockcraft docs](https://canonical-rockcraft.readthedocs-hosted.com/en/latest/explanation/rocks/).
-
-
-  
+ --- USAGE INFORMATION ---
+docker:
+  parameters:
+    - -p 2181:2181
+  access: Access your ZooKeeper server at `http://localhost:2181`.
+parameters:
+  - type: -e
+    value: 'TZ=UTC'
+    description: Timezone.
+  - type: -p
+    value: '2181:2181'
+    description: Expose ZooKeeper server on `localhost:2181`.
+  - type: -v
+    value: "/path/to/config/file:/etc/zookeeper/zoo.cfg"
+    description: Local ZooKeeper configuration file.
+  - type: -v
+    value: "zookeeperData:/var/lib/zookeeper/data"
+    description: > 
+      "Persist data in a docker volume named `zookeeperData`. "
+      "Make sure that the mount point is consistent with the configuration property `dataDir`.
+  - type: -v
+    value: "zookeeperLogData:/var/lib/zookeeper/data-log"
+    description: > 
+      "Persist data in a docker volume named `zookeeperLogData`. "
+      "Make sure that the mount point is consistent with the configuration property `dataLogDir`.
+debug:
+  text: |
+    ### Debugging
+    
+    To debug the container:
+    ```bash
+    docker logs -f <zookeeper-container>
+    ```
+    
+    To get an interactive shell:
+    ```bash
+    docker exec -it <zookeeper-container> /bin/bash
```

Comment:
@cjdcordeiro I tried to have this consistent with [this](https://github.com/canonical/oci-factory/pull/48/files#diff-9bb32735cb189a38cdbe3ee4c131c0a2d970adfa2c35290ebb9e478d480e7e98R49), but the Rock syntax

```
docker exec -it <zookeeper-container> exec /bin/bash 
```

did not work for me. It provided this output:

```
CI runtime exec failed: exec failed: unable to start container process: exec: "exec": executable file not found in $PATH: unknown
```

I believe it is expected, but I'm not sure why it works for the other image.

Reply:
hum weird. It works for me.

I took your rock from https://github.com/canonical/zookeeper-rock/actions/runs/8094163188, 

then `skopeo copy oci-archive:zookeeper_3.8.2_amd64.rock docker-daemon:test:latest` and then

```bash
$ docker run --rm -it test exec /bin/bash
2024-02-29T15:14:03.595Z [pebble] Started daemon.
2024-02-29T15:14:03.601Z [pebble] POST /v1/exec 3.14273ms 202
2024-02-29T15:14:03.604Z [pebble] GET /v1/tasks/1/websocket/control 2.84456ms 200
2024-02-29T15:14:03.604Z [pebble] GET /v1/tasks/1/websocket/stdio 40.835µs 200
root@ae43a10f8a7c:~# 
root@ae43a10f8a7c:~# 
root@ae43a10f8a7c:~# whoami
root
root@ae43a10f8a7c:~# 
```

can you share your steps?

Reply:
yes, but that's not what is in the documentation I linked. If you do:

```
docker run --rm zookeeper:latest
```

you start the zookeeper process. But you then want to debug something that requires to login in the container. Then I first find the container id with `docker ps`, and use the container id in the command

```
docker exec -it <container-id> exec /bin/bash
```

This throws an error 

```
OCI runtime exec failed: exec failed: unable to start container process: exec: "exec": executable file not found in $PATH: unknown
```

On the other hand, using the syntax

```
docker exec -it <container-id> /bin/bash
```

works fine.